diff --git a/lib/libzfs/libzfs_sendrecv.c b/lib/libzfs/libzfs_sendrecv.c
index 3e22edee9..43290f230 100644
--- a/lib/libzfs/libzfs_sendrecv.c
+++ b/lib/libzfs/libzfs_sendrecv.c
@@ -4888,7 +4888,8 @@ zfs_receive_one(libzfs_handle_t *hdl, int infd, const char *tosnap,
 		zfs_nicebytes(bytes, buf1, sizeof (buf1));
 		zfs_nicebytes(bytes/delta, buf2, sizeof (buf1));
 
-		(void) printf("received %s stream in %" ZFS_TIME_T_FORMAT "u seconds (%s/sec)\n",
+		(void) printf("received %s stream "
+		    "in %" ZFS_TIME_T_FORMAT "u seconds (%s/sec)\n",
 		    buf1, delta, buf2);
 	}
 
